subprojects/docs/build.gradle

```diff
@@ -617,6 +617,7 @@ tasks.named("docsTest") { task ->
             excludeTestsMatching "org.gradle.docs.samples.*.java-modules-*.sample"
             excludeTestsMatching "org.gradle.docs.samples.*.incubating-java-modules-*.sample"
             excludeTestsMatching 'org.gradle.docs.samples.*.snippet-java-library-module*.sample'
+            excludeTestsMatching 'org.gradle.docs.samples.*.snippet-testing-patch-module*.sample'
```

Comment:
❓ Why?

Reply:
The snippet under test is modular, so it cannot run on Java 8. This CL adds a test case to actually compile stuff, so it now starts failing on 8.

Reply:
Got it. Thanks 👍

Reply:
In general it would be good to leave a comment with each `exclueTestsMatching` like I've seen it done in other places.

Reply:
There's a comment at the beginning of the if statement :)

Reply:
Indeed there is :)